pkg/bot/lark.go

```diff
@@ -96,23 +96,23 @@ func NewLarkBot(c *config.Config) Bot {
 }
 
 // Execute commands sent by users
-func (l *LarkBot) Execute(e map[string]interface{}) {
+func (l *LarkBot) Execute(e map[string]interface{}) error {
 	event, err := accessAndTypeCastToMap(larkEvent, e)
 	if err != nil {
 		log.Errorf("Lark: AccessAndTypeCastToMap: %s", err.Error())
```

Comment:
Since we are returning an error now, we should log the error on the caller's side. We should just wrap and return error from here.